python_file_reader.py

```diff
@@ -0,0 +1,31 @@
+def twod_read_input(mylist, file_name): #returns text from given file in a 2d list
+    f = open(file_name, 'r')
+    file_data = f.read()
+    input_text_block = file_data.strip().split('\n')
+    for line in input_text_block:
+        row = []
+        for char in line:
+            row.append(char)
+        mylist.append(row)
+
+def oned_read_input(mylist, file_name): #returns text from given file in a 1d list
+    f = open(file_name, 'r')
+    file_data = f.read()
+    input_text_block = file_data.strip().split('\n')
```

Comment:
I think moving this into a separate function that both helper functions call would be a bit cleaner :+1:

Reply:
I have 1 in each so that a user can just copy paste either function as needed, since I haven't run into any problems that need both functions